Content.Server/Nuke/NukeComponent.cs

```diff
@@ -55,8 +55,8 @@ public sealed partial class NukeComponent : SharedNukeComponent
         /// <summary>
         ///     How long a user must wait to disarm the bomb.
         /// </summary>
-        [DataField("disarmDoafterLength")]
-        public float DisarmDoafterLength = 30.0f;
+        [DataField("disarmDoAfterLength")]
```

Comment:
```suggestion
        [DataField]
```
Datafield names are automatically converted from PascalCase to camelCase in yaml, so this is redundant
But I would probably just keep it as it is to prevent a breaking change for forks. Could you undo renaming this?